app/forms/idv/api_image_upload_form.rb

```diff
@@ -262,6 +273,15 @@ def validate_duplicate_images
           side: error_sides.length == 2 ? 'both' : error_sides[0], **extra_attributes,
         )
       end
+
+      if capture_result&.failed_selfie_image?(selfie_image_fingerprint)
+        errors.add(
+          :selfie, t('doc_auth.errors.doc.resubmit_failed_image'), type: :duplicate_image
+        )
+        analytics.idv_doc_auth_failed_image_resubmitted(
+          side: 'selfie', **extra_attributes,
+        )
+      end
```

Comment:
I think I'd prefer to have this above the `unless error_sides.empty?` on line 271.
- Currently (on `main`) we will either get one `idv_doc_auth_failed_image_resubmitted` event or zero.
- With this change we could get zero, one, or two `idv_doc_auth_failed_image_resubmitted` events.
- I think it would be better to continue to get one or zero of these, but change the possible values in the event?

Reply:
@charleyf 👋🏿  - if we continue with one or zero analytics events, do you have a suggestions on how we should define side? or the values you would change?

also, can you please share your concerns for having an additional analytics event just for selfie?

i could see the preference to move this code above `unless_error_sides.empty?` if we have only one or zero analytics events? 👍🏿

Reply:
Interesting points, looking at it again, I think the way you did this is best 👍 

- When I was looking at this earlier I was thinking that we might be interpreting the logs as one `idv_doc_auth_failed_image_resubmitted` event being roughly equal to one image failing upload.
- Looking again, I see that it's more likely we'd interpret one `idv_doc_auth_failed_image_resubmitted` event as being some number of failed doc images (one or two).
- Also, thinking about this again, changing an existing event is much more likely to cause log-querying problems than adding a new instance of that event. More confirmation for the implementation you chose.